src/aktualizr_secondary/update_agent_ostree.cc

```diff
@@ -78,18 +78,7 @@ data::ResultCode::Numeric OstreeUpdateAgent::install(const Uptane::Target& targe
 }
 
 data::InstallationResult OstreeUpdateAgent::applyPendingInstall(const Uptane::Target& target) {
-  if (!_ostreePackMan->rebootDetected()) {
-    // it should be removed from here, once we refactor the ostree package manager,
-    // e.g. _ostreePackMan->finalizeInstall() does this check/if
-    return data::InstallationResult(data::ResultCode::Numeric::kNeedCompletion,
-                                    "Reboot is required for the pending update application");
-  }
-
-  data::InstallationResult install_result = _ostreePackMan->finalizeInstall(target);
-  // it should be removed from here, once we refactor the ostree package manager,
-  // e.g. the pacman will reset/clear the flag by itself
-  _ostreePackMan->rebootFlagClear();
-  return install_result;
+  return _ostreePackMan->finalizeInstall(target);
```

Comment:
Not related to this PR really but according to good sources, identifiers starting with underscores should be avoided as they are reserved for "C++ implementations": https://stackoverflow.com/a/20496955/1931271

Reply:
It actually contradicts the google coding guidelines that we supposed to follow, I found out about it just after this change was merged. I will change this underscore naming to make it consistent with our coding style in another PR. 

But personally I prefer/like this code guideline https://www.appinf.com/download/CppCodingStyleGuide.pdf which says the following in regard of underscore usage:

"
```
Rule 30
Global names must not begin with one or two underscores (“_” or “__”).
This is forbidden by the C++ standard as such names are reserved for use by
the compiler.

Rule 31
With the exceptions of constants and enumeration values, there are no
underscores in names (after the first character of the name). 2
"

Rule 33
The name of a private or protected member variable begins with a single
underscore.3
 This is the only underscore in such a name (see Rule 31).


```

Reply:
Ah cool, this guide might have interesting insights as well.

In this case, however, even the C++ standard has something to say (2.10):

> In addition, some identifiers are reserved for use by C++implementations and shall not be used otherwise;no diagnostic is required.
— Each identifier that contains a double underscore__or begins with an underscore followed by an uppercase letter is reserved to the implementation for any use.
— Each identifier that begins with an underscore is reserved to the implementation for use as a name int he global namespace.

It doesn't exactly conflict as these are not in the global namespace and do not follow the underscore by an uppercase letter but that's quite close.